testsuites/CBLTester/CBL_Functional_tests/conftest.py

```diff
@@ -320,6 +320,9 @@ def params_from_base_suite_setup(request):
         log_info("Running without delta sync")
         persist_cluster_config_environment_prop(cluster_config, 'delta_sync_enabled', False)
 
+    # As cblite jobs run with on Centos platform, adding by default centos to environment config
+    persist_cluster_config_environment_prop(cluster_config, 'sg_platform', "centos", False)
```

Comment:
don't we have SG jobs for other platforms like ubuntu and macOS?

Reply:
> don't we have SG jobs for other platforms like ubuntu and macOS?

we have SG jobs which has  parameter for sg_platform for sg conftest file. This conftest is for CBL which we are not having sg-platform as a parameter and running default on centos. That is why I added default as centos.  For new changes of mac support needs sg_platform parameter in cluster configs 'reset' method expects to have sync-gateway path for x509 changes to work.  So, that mac support can work with x509 changes.